src/OpenTelemetry.Exporter.Jaeger/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
   Supported values: `udp/thrift.compact` and `http/thrift.binary` defined
   in the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/9a0a3300c6269c2837a1d7c9c5232ec816f63222/specification/sdk-environment-variables.md?plain=1#L129).
   ([#2914](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2914))
+* Change handling of `OTEL_EXPORTER_JAEGER_ENDPOINT` to require the path to post. Previous versions of this library would append `/api/traces` to the value specified in this variable, but now
```

Comment:
Please add a link to this pr or the issue as well.

Reply:
Thanks. I also linked to the PR that updated the spec.